css/index.css

```diff
@@ -1,3 +1,236 @@
+   License: none (public domain)
+*/
 
+html, body, div, span, applet, object, iframe,
+h1, h2, h3, h4, h5, h6, p, blockquote, pre,
+a, abbr, acronym, address, big, cite, code,
+del, dfn, em, img, ins, kbd, q, s, samp,
+small, strike, strong, sub, sup, tt, var,
+b, u, i, center,
+dl, dt, dd, ol, ul, li,
+fieldset, form, label, legend,
+table, caption, tbody, tfoot, thead, tr, th, td,
+article, aside, canvas, details, embed,
+figure, figcaption, footer, header, hgroup,
+menu, nav, output, ruby, section, summary,
+time, mark, audio, video {
+    margin: 0;
+    padding: 0;
+    border: 0;
+    font-size: 100%;
+    font: inherit;
+    vertical-align: baseline;
+}
+/* HTML5 display-role reset for older browsers */
+article, aside, details, figcaption, figure,
+footer, header, hgroup, menu, nav, section {
+    display: block;
+}
+body {
+    line-height: 1;
+}
+ol, ul {
+    list-style: none;
+}
+blockquote, q {
+    quotes: none;
+}
+blockquote:before, blockquote:after,
+q:before, q:after {
+    content: '';
+    content: none;
+}
+table {
+    border-collapse: collapse;
+    border-spacing: 0;
+}
+
+/* Set every element's box-sizing to border-box */
+* {
+    box-sizing: border-box;
+}
+
+html, body {
+    height: 100%;
+    font-family: 'Titillium Web', sans-serif;
+}
+
+h1, h2, h3, h4, h5 {
+    font-family: 'Bangers', cursive;
+    letter-spacing: 1px;
+    margin-bottom: 15px;
+}
+/*post Meyer, code starts here*/
+
+/*Great Idea Index*/
+.index-page{
+    width: 1000px;
+    margin: auto;
+    display: flex;
+    flex-direction: column;
+}
+.header{
+    width: 100%;
+    display: flex;
+    align-items: center;
+    justify-content: space-evenly;
+    margin-top: 20px;
+}
+.nav-panel{
+    width: 100%;
+    display: flex;
+    justify-content: space-evenly;
+    align-items: center;
+}
+.nav-panel a {
+    text-decoration: none;
+    color: grey;
+    margin: 20px 20px 0 20px;
+}
+.top-section{
+    display: flex;
+    width: 100%;
+    margin-top: 30px;
+    margin-bottom: 30px;
+}
+.sloganDiv{
+    display: flex;
+    flex-flow: column nowrap;
+    width: 50%;
+    align-items: center;
+    justify-content: center;
+}
+.accentImage{
+    width: 50%;
+}
+.slogan {
+    font-size: 6rem;
+    text-align: center;
+}
+.index-button{
+    width: 150px;
+    height: 40px;
+    border-radius: 10px;
+    font-size: 11pt;
+    background: snow;
+    border: 2px solid black;
+}
+.divSetOne{
+    width: 100%;
+    display: flex;
+    flex-direction: row;
+    margin-top: 25px;
+    margin-bottom: 25px;
+    align-items: space-between;
+}
+.middle-img{
+    width: 100%;
+}
+.divSetTwo{
+    width: 100%;
+    display: flex;
+    flex-direction: row;
+    margin-top: 25px;
+    margin-bottom: 25px;
+    align-items: space-between;
+}
+.bottom-section{
+    margin-bottom: 40px;
+    margin-top: 40px;
+}
+footer {
+    text-align: center;
+    margin-bottom: 40px;
+}
+
+/*Begin Mobile*/
+@media only screen
+and (max-width: 599px)
+and (min-width: 300px)
```

Comment:
For this part, by setting a min-width property you have removed these CSS properties for widths below 300px.  Because of that and that you did the same for tablet, for screen widths below 300px it actually reverts to the desktop view.  I would consider removing the min-width at this section here.